pkg/controllers/restore/controller.go

```diff
@@ -538,7 +552,9 @@ func (h *handler) createFromDependencyGraph(ownerToDependentsList map[string][]r
 		} else {
 			resourceData = objFromBackupCR.clusterscopedResourceInfoToData[currResourceInfo]
 		}
-		if err := h.restoreResource(currResourceInfo, resourceData, objFromBackupCR.resourcesWithStatusSubresource[curr.GVR.String()]); err != nil {
+		target := fmt.Sprintf("%s.%s", currResourceInfo.GVR.Resource, currResourceInfo.GVR.GroupVersion().String())
+		hasSubStatus := slice.ContainsString(crdWithSubStatus, target)
```

Comment:
Why isn't this an exact match over the substring search? Not opposed, just curious.

Reply:
The method `slice.ContainsString` is actually doing an exact match internally:
```
func ContainsString(slice []string, item string) bool {
	for _, j := range slice {
		if j == item {
			return true
		}
	}
	return false
}
```